bodhi/server/services/releases.py

```diff
@@ -332,3 +350,47 @@ def save_release(request):
     request.db.flush()
 
     return r
+
+
+@critpath_packages.put(schema=bodhi.server.schemas.UpdateCriticalPackages,
+                       permission='edit', renderer='json',
+                       error_handler=bodhi.server.services.errors.json_handler,
+                       validators=(colander_body_validator, validate_release, validate_packages)
+                       )
+def add_critpath_packages(request):
+    """
+    Add a list of critical path packages to a specific release.
+
+    Args:
+        request (pyramid.request): The current request.
+    Returns:
+        dict: The dictionary {'status': 'success'}.
+    """
+    data = request.validated
+    release = data['release']
+
+    release.add_critpath_pkgs(data['packages'])
```

Comment:
We should add a validator for packages that ensures they exist, creates them if they don't, and turns them into Package objects instead of strings.